index.js

```diff
@@ -109,6 +129,21 @@ module.exports = (opts = {}) => {
         }
       }
     },
+    DeclarationExit: (declaration, { postcss }) => {
+      if (!declaration.value.startsWith('light-dark(')) return
```

Comment:
I am not sure that all decl should start from `ligh-dark()`.

What about: `border-color: black light-dark(black, white)`